hydromt_fiat/api/data_types.py

```diff
@@ -10,8 +10,8 @@ class ExtractionMethod(str, Enum):
 
 
 class Units(str, Enum):
-    m = "m"
-    ft = "ft"
+    m = "meter"
```

Comment:
"meter" should be "meters".

And isn't it more standard to use the same name and string? Like:
`meters = "meters"`